core/lib/Cardano/Address.hs

```diff
@@ -237,3 +220,15 @@ invariantNetworkTag limit (NetworkTag num)
       ++ show num
       ++ ", but expected to be less than "
       ++ show limit
+
+newtype AddressVerificationKeyHash = AddressVerificationKeyHash { unAddressVerificationKeyHash :: ByteString }
+instance ToBech32 AddressVerificationKeyHash where
+    bech32 (AddressVerificationKeyHash kh) = bech32With CIP5.addr_vkh kh
+instance FromBech32 AddressVerificationKeyHash where
+    fromBech32 = fromBech32With CIP5.addr_vkh AddressVerificationKeyHash
+
+newtype StakeVerificationKeyHash = StakeVerificationKeyHash { unStakeVerificationKeyHash :: ByteString }
```

Comment:
```suggestion
newtype StakeVerificationKeyHash =
    StakeVerificationKeyHash { unStakeVerificationKeyHash :: ByteString }
```

https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#limit-line-length-to-80-characters